lib/workers/pr/changelog/release-notes.ts

```diff
@@ -130,6 +131,17 @@ function sectionize(text: string, level: number): string[] {
   return result;
 }
 
+function isUrl(url: string): boolean {
+  try {
+    return !!URL.parse(url).hostname;
+  } catch (err) {
+    // istanbul ignore next
+    logger.warn({ err }, `Error parsing ${url} in URL.parse`);
```

Comment:
```suggestion
    logger.debug({ err }, `Error parsing ${url} in URL.parse`);
```

This is not something people should be concerned about

Reply:
okay!